### PR TITLE
Contributing Guide

### PR DESCRIPTION
**Pull request by arianerocha**
This PR adds the first version of the Contributing Guide.

### CODE REVIEW

File: docs/development/contribute.md [hunk 1]
```diff
@@ -5,4 +5,111 @@ sidebar_label: Contribute
...
+
+ - CI is passing.
+ - It was approved by at least two maintainers. If it was a maintainer who opened the PR, only extra approval is required.
+ - No changes requested.
+ - It is updated with the current branch main code.
+
+## Get in Touch
+
+  - Sumé Community on [Discord](https://discord.com/invite/aaJzjBunTZ), in case you need help or want to make suggestions.
+  - Look for / ask questions at [Stack Overflow] ([https://stackoverflow.com/questions/tagged/sumelms](https://stackoverflow.com/questions/tagged/sumelms)).
```
Comment (rluders) [inline docs/development/contribute.md:114]: Este link está quebrado, tem um espaço entre `[]` e `()`

File: docs/development/contribute.md [hunk 2]
```diff
@@ -5,4 +5,111 @@ sidebar_label: Contribute
...
+  - Chat with the community on the [Reddit] forum ([https://new.reddit.com/r/sumelms/](https://new.reddit.com/r/sumelms/)).
```
Comment (rluders) [inline docs/development/contribute.md:115]: mesma coisa aqui.

**Review by rluders — CHANGES_REQUESTED** [review]

File: docs/development/contribute.md [hunk 3]
```diff
@@ -1,8 +1,149 @@
...
+
+### Feature requests
+
+Feature requests are welcome. We are open to new ideas and proposed changes to comply with the user needs.
+
+If you want to contribute, open an issue in the respective repository on [GitHub](https://github.com/sumelms), describe as much as possible what you would like to see in Sumé, why you think it would be necessary, and how it should work.
+
+## Project Management (maintainers only)
+
+In order to keep things organized all our repositories is able to have their own issues, but you can found our [project board with all the tickets here](https://github.com/orgs/sumelms/projects/1).
```
Comment (rluders) [inline docs/development/contribute.md:45]: To keep things organized all our repositories can have their issues, but you can found our [project board with all the tickets here](https://github.com/orgs/sumelms/projects/1).

File: docs/development/contribute.md [hunk 4]
```diff
@@ -1,8 +1,149 @@
...
+### Ticket classification
+
+#### Epics and User Stories
+
+- **Epics** are issues that to be solved need to be split into small tasks called *user stories*.
+- **User Stories** are issues that to be solved need to be split into small and executable issues. 
+
+This type of issues we call them "planning issues", 'cause they are the ones that can control and track the execution of the other issues.
+
+These tasks should, in fact, hold the necessary amount of the strategic view of its goal as well the links to other documents and related issues - nothing more, nothing less.
```
Comment (rluders) [inline docs/development/contribute.md:56]: These tasks should hold the necessary amount of the strategic view of its goal as well the links to other documents and related issues - nothing more, nothing less.

File: docs/development/contribute.md [hunk 5]
```diff
@@ -1,8 +1,149 @@
...
+
+In order to help you to identify an Epic and a User Story, remember:
```
Comment (rluders) [inline docs/development/contribute.md:58]: To help you to identify an Epic and a User Story, remember:

File: docs/development/contribute.md [hunk 6]
```diff
@@ -1,8 +1,149 @@
...
+- Do not generate any PR
+- Cannot be directly executed
+
+#### Issues
+
+Issues are executable tasks. If they are related to new features or enhancements, it is pretty common that they are part of a User Story, also these small and executable tasks can be bugs, in this case, it is pretty normal that they are **not** related to any User Story.
+
+In order to help you to identify an Issue, remember:
+
+- Can be assign to someone
```
Comment (rluders) [inline docs/development/contribute.md:71]: assigned

File: docs/development/contribute.md [hunk 7]
```diff
@@ -1,8 +1,149 @@
...
+- **Epics** are issues that to be solved need to be split into small tasks called *user stories*.
+- **User Stories** are issues that to be solved need to be split into small and executable issues. 
+
+This type of issues we call them "planning issues", 'cause they are the ones that can control and track the execution of the other issues.
+
+These tasks should, in fact, hold the necessary amount of the strategic view of its goal as well the links to other documents and related issues - nothing more, nothing less.
+
+In order to help you to identify an Epic and a User Story, remember:
+
+- Cannot be assign to anybody
```
Comment (rluders) [inline docs/development/contribute.md:60]: assigned

File: docs/development/contribute.md [hunk 8]
```diff
@@ -1,8 +1,149 @@
...
+- N**ormally** lives at the [main repository](https://github.com/sumelms/sumelms)
+- Do not generate any PR
+- Cannot be directly executed
+
+#### Issues
+
+Issues are executable tasks. If they are related to new features or enhancements, it is pretty common that they are part of a User Story, also these small and executable tasks can be bugs, in this case, it is pretty normal that they are **not** related to any User Story.
+
+In order to help you to identify an Issue, remember:
```
Comment (rluders) [inline docs/development/contribute.md:69]: To help you to identify an Issue, remember:

File: docs/development/contribute.md [hunk 9]
```diff
@@ -1,8 +1,149 @@
...
+
+- Can be assign to someone
+- Lives in task-specific repository
```
Comment (rluders) [inline docs/development/contribute.md:72]: Lives in the task-specific repository

File: docs/welcome.md [hunk 10]
```diff
@@ -8,9 +8,12 @@ slug: /
 ## About
 
 
-
 ## Features
 
+## Contribution
+
+Please check the instructions on [Contributing Guide](/docs/development/contribute) before making a pull request.
```
Comment (rluders) [inline docs/welcome.md:15]: Não é melhor linkar para a página no website?

**Review by rluders — CHANGES_REQUESTED** [review]

File: docs/welcome.md (hunk 10 above)
Comment (arianerocha) [reply docs/welcome.md:15]: Não entendi, já não está?

File: docs/development/contribute.md (hunk 3 above)
Comment (arianerocha) [reply docs/development/contribute.md]: done

File: docs/development/contribute.md (hunk 4 above)
Comment (arianerocha) [reply docs/development/contribute.md]: Done!

File: docs/development/contribute.md (hunk 5 above)
Comment (arianerocha) [reply docs/development/contribute.md]: Done

File: docs/development/contribute.md (hunk 6 above)
Comment (arianerocha) [reply docs/development/contribute.md]: Done

File: docs/development/contribute.md (hunk 8 above)
Comment (arianerocha) [reply docs/development/contribute.md]: Done

File: docs/development/contribute.md (hunk 7 above)
Comment (arianerocha) [reply docs/development/contribute.md]: Done

File: docs/development/contribute.md (hunk 9 above)
Comment (arianerocha) [reply docs/development/contribute.md]: Done

File: docs/welcome.md (hunk 10 above)
Comment (rluders) [reply docs/welcome.md:15]: Não deveria estar `https://...`?

Comment (rluders) [reply docs/welcome.md:15]: Ignore. Acabei de perceber que tá certo.

**Review by rluders — APPROVED** [review]

**Pull request merged**